infra/aws/terraform/prow-build-cluster/resources/monitoring/node-exporter/ds.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: node-exporter
+  namespace: monitoring
+  labels:
+    app: node-exporter
+spec:
+  selector:
+    matchLabels:
+      app: node-exporter
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
+  template:
+    metadata:
+      labels:
+        app: node-exporter
+    spec:
+      serviceAccountName: node-exporter
+      securityContext:
+        runAsNonRoot: true
+        runAsUser: 65534
+      containers:
+        - name: node-exporter
+          image: public.ecr.aws/bitnami/node-exporter:1.4.0
```

Comment:
Is there a specific reason why we don't use the official image? Can we document it somewhere?

Reply:
No specific reason. I'll replace it with `quay.io/prometheus/node-exporter:v1.5.0`